docs/services/slam/cartographer/_index.md

```diff
@@ -293,18 +293,30 @@ Adjust these parameters to fine-tune the algorithm `cartographer` utilizes in as
 | Parameter Mode | Description | Inclusion | Default Value | Notes |
 | -------------- | ----------- | --------- | ------------- | ----- |
 | `mode` | `2d` | **Required** | None | |
-| `optimize_every_n_nodes` | How many trajectory nodes are inserted before the global optimization is run. | Optional | `3` | |
-| `num_range_data` | Number of measurements in each submap. | Optional | `100` | |
-| `missing_data_ray_length` | Replaces the length of ranges that are further than `max_range` with this value. | Optional | `25` | Nominally set to max length. |
+| `optimize_every_n_nodes` | How many trajectory nodes are inserted before the global optimization is run. | Optional | `3` | Setting to `0` will disable global SLAM, using local SLAM only. |
+| `num_range_data` | Number of measurements in each submap. | Optional | `30` | |
+| `missing_data_ray_length` | Replaces the length of ranges that are further than `max_range` with this value. | Optional | `25` | Should be the same as `max_range`. |
```

Comment:
From what I've seen there might be some applications where it's worth setting to a bit lower than max_range. It might be better to change it to "Typically the same as max_range" to indicate this?

Reply:
That phrasing sounds good to me!